src/common/plugin/pluginState.ts

```diff
@@ -142,7 +135,8 @@ export function makePluginState(settings: PluginStateSettings): PluginState {
   const refreshServers = async (): Promise<void> => {
     let serverList = defaultServers
 
-    if (!disableFetchingServers)
+    // Does this logic seem backwards?
+    if (enableCustomServers)
```

Comment:
Oops, looking at the diff, I see I deleted the `!`, which is indeed why the logic is backwards now. Need to fix that.